src/SFA.DAS.ProviderRequestApprenticeTraining.Application/Commands/CreateProviderResponseEmployerRequest/CreateProviderResponseEmployerRequestCommand.cs

```diff
@@ -0,0 +1,11 @@
+﻿using MediatR;
+
+namespace SFA.DAS.ProviderRequestApprenticeTraining.Application.Commands.CreateProviderResponseEmployerRequest
+{
+    public class CreateProviderResponseEmployerRequestCommand : IRequest<bool>
+    {
+        public long Ukprn { get; set; }
+        public List<Guid> EmployerRequestIds { get; set; }
+        public int ProviderResponseStatus { get; set; }
```

Comment:
ProviderResponseStatus - that doesn't appear in the outer or the inner api?

Reply:
REmoved